index.js

```diff
@@ -1,6 +1,140 @@
-exports.build = function (options) {
-  // call flatpak build-init
-  // copy in files
-  // flatpak build-finish
-  console.log("NOT IMPLEMETED");
+'use strict';
+
+var _ = require('lodash');
+var path = require('path');
+var util = require('util');
+
+var promise = require('bluebird');
+var exec = promise.promisify(require('child_process').exec);
+var copy = promise.promisify(require('fs-extra').copy);
```

Comment:
You could use `promise.promisifyAll` here or import one of the fs-extra drop-in replacements